gui/main.py

```diff
@@ -0,0 +1,218 @@
+"""
```

Comment:
Sorry if I'm missing something but is that `gui/main.py` related/wanted in that PR ? Thanks !